session_pool.go

```diff
@@ -53,7 +52,8 @@ type pureSession struct {
 	sessPool   *SessionPool
 	returnedAt time.Time // the timestamp that the session was created or returned.
 	timezoneInfo
-	spaceName string
+	spaceName  string
+	enableHttp bool
```

Comment:
This is redundant. But I will let it merge for now. Please remove it in the following PR.